modules/nixos/software/general.nix

```diff
@@ -0,0 +1,7 @@
+{ pkgs, ... }:
+
+{
+  environment.systemPackages = with pkgs; [
+    tmux mosh
+  ];
```

Comment:
these must be guarded by an option and default to false.

Reply:
I'm going to leave it as is for now. I should move Tmux in a similar way to NeoVim.